Python-packages/covidcast-py/covidcast/plotting.py

```diff
@@ -286,21 +289,22 @@ def _plot_bubble(ax: axes.Axes, data: gpd.GeoDataFrame, geo_type: str, **kwargs:
     ax.legend(frameon=False, ncol=8, loc="lower center", bbox_to_anchor=(0.5, -0.1))
 
 
-def _plot_background_states(figsize: tuple) -> tuple:
+def _plot_background_states(figsize: tuple, ax: axes.Axes = None) -> axes.Axes:
     """Plot US states in light grey as the background for other plots.
 
     :param figsize: Dimensions of plot.
```

Comment:
So figsize is ignored if axes are provided? Should it also be optional?

Reply:
added